karate-tests/src/test/java/patients/healthcareWorker/createPatient/postPatient.feature

```diff
@@ -44,7 +44,8 @@ Background:
   * url baseURL
 
 Scenario: Post patient - new patient
-  * def familyName = "Karate-test-" + utils.randomString(7)
+  # Don't change the familyName
+  * def familyName = "ToRemove"
```

Comment:
Are we breaking some expectations if we're no longer using a random string in the family name? (would that have an impact on subsequent patient creation requests and the corresponding matching?)